### DISCUSSION TITLE
Pipeline expense-terraform

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1530] [Q&A] [answered]
hi,
 when i build the pipeline its having error that Unable to find Jenkinsfil even i created this jenkinsfile
 
![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/9289eed2-d8c4-4e54-9ede-96c68238556f)
![Capture1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/d905675f-d69a-435e-9701-f800430f7074)

**Comment by sreedharm07** [comment, marked as answer]
hi @aambadkar 
give capital J for jenkins file **Jenkinsfile**
